website/docs/reference/command-menu.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Command Menu
+---
+
+:::note Availability
```

Comment:
Just for context: I'd like to begin to standardize how the plan/version availability info looks on all the pages. My dream is to have labels in the future but for now, this is what I've been going for